srv/modules/runners/orderednodes.py

```diff
@@ -15,6 +15,16 @@ def _preserve_order_sorted(seq):
     seen = set()
     return [x for x in seq if x not in seen and not seen.add(x)]
 
+def only(cluster='ceph', role=None):
```

Comment:
Sorry of course the function can return a list of a certain role. Though I believe the select runner can do this as well via ```select.attr```?